components/widgets/icons/fa_next_icon.tsx

```diff
@@ -25,7 +24,7 @@ export default class NextIcon extends React.PureComponent {
                 {(title) => (
                     <i
                         className={className}
-                        title={title}
+                        title={title as string}
                     />
                 )}
```

Comment:
Some other files have this function in them, but nothing else to change. Should I edit those to include {title as string} or is it fine?